Omise/Client.cs

```diff
@@ -7,13 +7,14 @@ namespace Omise
     public class Client
     {
         // Default to the latest API Version.
-        private string apiVersion = "2017-11-02";
```

Comment:
We can't change version. It'll break since fields in each version are different.

Reply:
The 2017 version uses a different syntax, so some significant changes have been made.